mmpose/datasets/transforms/formatting.py

```diff
@@ -99,7 +99,8 @@ class PackPoseInputs(BaseTransform):
         'keypoint_x_labels': 'keypoint_x_labels',
         'keypoint_y_labels': 'keypoint_y_labels',
         'keypoint_weights': 'keypoint_weights',
-        'instance_coords': 'instance_coords'
+        'instance_coords': 'instance_coords',
+        'keypoints_visible': 'keypoints_visible',
```

Comment:
Could you please support `transformed_keypoints_visible` in https://github.com/open-mmlab/mmpose/blob/0796d77361201a862e9341d512b5445751df8832/mmpose/datasets/transforms/formatting.py#L156 
and https://github.com/open-mmlab/mmpose/blob/0796d77361201a862e9341d512b5445751df8832/mmpose/datasets/transforms/common_transforms.py#L935
because keypoint visibility will be used in `browse_dataset.py`